jerry-core/ecma/builtin-objects/ecma-builtin-json.cpp

```diff
@@ -41,21 +41,22 @@
 
 /*
  * FIXME:
- *       Replace usage of isalpha and isdigit functions in the module with lit_char helpers and remove the functions.
+ *       Replace usage of ecma_builtin_json_isalpha and ecma_builtin_json_isdigit functions in the module with lit_char
+ *       helpers and remove the functions.
  *
  *       Related issue: #424
  */
 
 /** Checks for an alphabetic character. */
 static int
-isalpha (int c)
+ecma_builtin_json_isalpha (int c)
```

Comment:
There are a bunch of similar functions in lit-char-helpers.cpp such as lit_char_is_decimal_digit. Why don't we move everything to there?